client/src/game/data/variantsInit.ts

```diff
@@ -24,7 +24,7 @@ interface VariantJSON {
 type VariantEntryIterable = Iterable<[keyof (typeof variantsJSON), VariantJSON]>;
 
 export default (COLORS: Map<string, Color>, SUITS: Map<string, Suit>, START_CARD_RANK: number) => {
-  const VARIANTS: Map<string, Variant> = new Map();
+  const VARIANTS: Map<string, Variant> = new Map<string, Variant>();
```

Comment:
you can kill the `: Map<string, Variant>` here

Reply:
@Zamiell There's a lint rule for that. Should I try it out? I feel like it might be a bit noisy 
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-inferrable-types.md